### PR TITLE
Move examples

### PR DESCRIPTION
**Pull request by seanturner**
Moved the examples to an Appendix.

### CODE REVIEW

**Review by bwesterb — APPROVED** [review]

File: draft-ietf-lamps-kyber-certificates.md [hunk 1]
```diff
@@ -253,20 +242,8 @@ The fields in SubjectPublicKeyInfo have the following meaning:
...
-  finalizing [DRAFTFIPS203].
-</aside>
-
-~~~
-  -----BEGIN PUBLIC KEY-----
-  TODO insert example public key
-  -----END PUBLIC KEY-------
-~~~
-
+{example-public} contains an example of an id-alg-ml-kem-768 public key
```
Comment (danvangeest) [inline draft-ietf-lamps-kyber-certificates.md:245]:
```suggestion
{{example-public}} contains an example of an id-alg-ml-kem-768 public key
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 2]
```diff
@@ -357,6 +305,9 @@ prior example, the textual encoding defined in {{RFC7468}} is used:
   key pair and widest ability to import the key.
 </aside>
 
+{example-private} contains an example of an id-alg-ml-kem-768 private key
```
Comment (danvangeest) [inline draft-ietf-lamps-kyber-certificates.md:308]:
```suggestion
{{example-private}} contains an example of an id-alg-ml-kem-768 private key
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 3]
```diff
@@ -377,6 +328,51 @@ This document will have some IANA actions.
 
 --- back
 
+# Examples {#examples}
+
+This appendix contains examples of ML-KEN public keys, private keys and certificates.
```
Comment (danvangeest) [inline draft-ietf-lamps-kyber-certificates.md:333]:
```suggestion
This appendix contains examples of ML-KEM public keys, private keys and certificates.
```

File: draft-ietf-lamps-kyber-certificates.md (hunk 2 above)
Comment (danvangeest) [inline draft-ietf-lamps-kyber-certificates.md:308]: Appendix says it's ML-KEM-512, here says 768.  Which one?  No actual examples to verify this against :)

File: draft-ietf-lamps-kyber-certificates.md (hunk 1 above)
Comment (danvangeest) [inline draft-ietf-lamps-kyber-certificates.md:245]: Appendix says it's ML-KEM-512, here says 768.  Which one?  No actual examples to verify this against :)

File: draft-ietf-lamps-kyber-certificates.md [hunk 4]
```diff
@@ -377,6 +328,51 @@ This document will have some IANA actions.
 
 --- back
 
+# Examples {#examples}
+
+This appendix contains examples of ML-KEM public keys, private keys and certificates.
+
+## Example Public Key {#example-public}
+
+The following is an example of a ML-KEM-512 public key:
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:337]:
```suggestion
The following is an example of a ML-KEM-768 public key:
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 5]
```diff
@@ -377,6 +328,51 @@ This document will have some IANA actions.
...
+
+~~~
+  -----BEGIN PUBLIC KEY-----
+  TODO insert example public key
+  -----END PUBLIC KEY-------
+~~~
+
+## Example Private Key {#example-private}
+
+The following is an example of a ML-KEM-512 private key:
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:347]:
```suggestion
The following is an example of a ML-KEM-768 private key:
```

Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:347]:
```suggestion
The following is an example of a ML-KEM-768 private key:
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 6]
```diff
@@ -377,6 +328,51 @@ This document will have some IANA actions.
...
+
+The following is an example of a ML-KEM-768 private key:
+
+~~~
+  -----BEGIN PRIVATE KEY-----
+  TODO insert example private key
+  -----END PRIVATE KEY-------
+~~~
+
+The following example, in addition to encoding the ML-KEM-512 private key,
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:355]:
```suggestion
The following example, in addition to encoding the ML-KEM-768 private key,
```

File: draft-ietf-lamps-kyber-certificates.md (hunk 1 above)
Comment (seanturner) [reply draft-ietf-lamps-kyber-certificates.md]: Yep should be 768!

File: draft-ietf-lamps-kyber-certificates.md (hunk 2 above)
Comment (seanturner) [reply draft-ietf-lamps-kyber-certificates.md]: Yep 768!

**Pull request merged**